### PR TITLE
Post page fixes

### PR DESCRIPTION
**Pull request by DoctorDib**
Main Changes:
 - Fixed overflow issue with vin by moving it into a dialog
 - Updated the comment input box to 4 rows with max of 10 rows.
 - Updated the comments wider
 - Updated Copyright to "Lost My Wheels 2020"
 - Updated default vehicle status to enum STOLEN (0)
 - Added formatting for "category" field in the Specifications table.
 - Added max size for image carousel (including thumnail tray) for a cleaner look.

### CODE REVIEW

File: client/src/Components/VehicleInfo/index.tsx [hunk 1]
```diff
@@ -19,37 +29,99 @@ interface IVehicleInfoProps {
...
-        </Grid>
-    </Grid>
-));
-
-const formatFeatures = (features: Array<string>) => features.map((damage: string) => (
-    <ListItem key={`damages - ${damage}`}>
-        <ListItemText primary={damage} />
-    </ListItem>
-));
+const formatFeatures = (features: Array<string>) =>
```
Comment (BenRGH) [inline client/src/Components/VehicleInfo/index.tsx]: needs return type

File: client/src/Components/VehicleInfo/index.tsx [hunk 2]
```diff
@@ -19,37 +29,99 @@ interface IVehicleInfoProps {
...
+    features.map((damage: string) => (
+        <ListItem key={`damages - ${damage}`}>
+            <ListItemText primary={damage} />
+        </ListItem>
+    ));
+
+const vinInformationPopup = (vin: string) => (
```
Comment (BenRGH) [inline client/src/Components/VehicleInfo/index.tsx]: needs return type

File: client/src/Components/VehicleInfo/index.tsx [hunk 3]
```diff
@@ -19,37 +29,99 @@ interface IVehicleInfoProps {
...
+    const capitalizeFirstLetter = (value: string) => {
+        if (value === undefined) {
+            return '';
+        }
+        return value.toLowerCase().charAt(0).toUpperCase() + value.toLowerCase().slice(1);
     };
 
-    const foundConfirmationResponse = () => {
-        setOpen(false);
+    const formatInfoValues = (key, value) => {
```
Comment (BenRGH) [inline client/src/Components/VehicleInfo/index.tsx:61]: args need type

File: client/src/Components/VehicleInfo/index.tsx [hunk 4]
```diff
@@ -19,37 +29,99 @@ interface IVehicleInfoProps {
...
-    const handleOpen = () => {
-        setOpen(true);
+    const handleOpen = () => setOpen(true);
+    const handleVinOpen = () => setOpenVin(true);
+    const handleVinClose = () => setOpenVin(false);
+    const foundConfirmationResponse = () => setOpen(false);
+
+    const { owner, vehicle } = props;
+
+    const capitalizeFirstLetter = (value: string) => {
```
Comment (BenRGH) [inline client/src/Components/VehicleInfo/index.tsx]: needs return type and moving to helpers

File: client/src/Components/VehicleInfo/index.tsx [hunk 5]
```diff
@@ -19,37 +29,99 @@ interface IVehicleInfoProps {
...
+        console.log(value);
+        if (key === 'category') {
+            return capitalizeFirstLetter(VehicleCategoryEnum[value]);
+        }
+
+        return value;
     };
 
-    const { owner, vehicle } = props;
+    const formatInfo = (data: IVehicleInfo) =>
```
Comment (BenRGH) [inline client/src/Components/VehicleInfo/index.tsx:71]: this might be a good candidate for useMemo

File: client/package.json [hunk 6]
```diff
@@ -51,30 +68,22 @@
...
     "eslint-plugin-react": "^7.21.4",
     "eslint-plugin-react-hooks": "^4.1.2",
+    "html-webpack-plugin": "^4.5.0",
     "jest": "^26.2.2",
-    "live-server": "^1.2.1",
     "prettier": "^2.0.5",
     "snowpack": "^2.14.0",
-    "typescript": "^4.0.0"
+    "typescript": "^4.0.0",
+    "webpack": "^5.1.0"
```
Comment (BenRGH) [inline client/package.json:87]: did you remove this webpack in a later pr?

File: client/src/Components/VehicleInfo/index.tsx (hunk 5 above)
Comment (DoctorDib) [reply client/src/Components/VehicleInfo/index.tsx:71]: I've never seen or used "useMemo" before

File: client/src/Components/VehicleInfo/index.tsx (hunk 3 above)
Comment (DoctorDib) [reply client/src/Components/VehicleInfo/index.tsx:61]: Fixed in a later PR

File: client/package.json (hunk 6 above)
Comment (DoctorDib) [reply client/package.json:87]: I think so, if not we'll go through it when we do the project code view at the end

File: client/src/Components/VehicleInfo/index.tsx (hunk 5 above)
Comment (DoctorDib) [reply client/src/Components/VehicleInfo/index.tsx:71]: As discussed, we're using useEffect in a later PR, we can switch over to useMemo whenever we make the fields was dynamic, right now it's only displaying static one time load data.

Comment (BenRGH) [reply client/src/Components/VehicleInfo/index.tsx:71]: Look into it, third most useful hook imo

File: client/src/Components/VehicleInfo/index.tsx [hunk 7]
```diff
@@ -29,14 +30,14 @@ interface IVehicleInfoProps {
     vehicle: IVehicleInfo;
 }
 
-const formatFeatures = (features: Array<string>) =>
+const formatFeatures = (features: Array<string>):ReadonlyArray<React.ReactElement> =>
```
Comment (BenRGH) [inline client/src/Components/VehicleInfo/index.tsx:33]: Do you want the return from this to be readonly?

Comment (DoctorDib) [reply client/src/Components/VehicleInfo/index.tsx:33]: Yeah, it's only used once

**Pull request merged**